templates/docs/layouts/fluid-breakout.md

```diff
@@ -0,0 +1,103 @@
+
+The wrapper aims to align as much as possible with the 12 column grid. On smaller screens, than is limited to ensuring the padding of both layouts match. Once the window width becomes large enough, the left edges of the regular 12 column grid and the central column of the fluid breakout layout also align. For this to be possible, two conditions need to be met:
+
+- <p>the central area's width needs to match the 12 column grid's width (we achieve this by setting the central area's max-width to the same value as the 12 column grid's max-width)</p>
+- <p>The window width is larger than the combined width of the 3 columns of the layout (`$l-fluid-breakout-max-width + 2 * $l-fluid-breakout-aside-width`)</p>
+
+#### Aside
+
+The aside is optional. When present, the order of the aside can be changed from before to after the main area by re-arranging the markup. Depending on the screen width, that would place it above / below (on screens smaller than `$breakpoint--large`) or to the left / right on larger screens.
+
+An aside to the left, main area to the right:
+
+<div class="embedded-example"><a href="/docs/examples/layouts/fluid-breakout/fluid-breakout-left-aside/" class="js-example">
+View example of he fluid breakout layout with a left aside
+</a></div>
+
+Or vice versa:
+
+<div class="embedded-example"><a href="/docs/examples/layouts/fluid-breakout/fluid-breakout-right-aside/" class="js-example">
+View example of he fluid breakout layout with a right aside
+</a></div>
+
+#### Main area
+
+The main area occupies the whole page on screens smaller than `$breakpoint-large`, then on larger screens allows a choice between spanning 2 or 3 of the layout's columns.
+
+Use the class `l-fluid-breakout__main` to span 2 columns:
+
+<div class="embedded-example"><a href="/docs/examples/layouts/fluid-breakout/fluid-breakout-left-aside/" class="js-example">
+View example of he fluid breakout layout without an aside
+</a></div>
+
+Use the class `l-fluid-breakout__full` to span all 3 columns:
+
+<div class="embedded-example"><a href="/docs/examples/layouts/fluid-breakout/fluid-breakout-full/" class="js-example">
+View example of he fluid breakout layout without an aside and a main column spanning all 3 columns:
+</a></div>
+
+In the absence of an aside, add the `.no-aside` class to the main area, to ensure it starts from the central column of the layout:
+
+<div class="embedded-example"><a href="/docs/examples/layouts/fluid-breakout/fluid-breakout-no-aside/" class="js-example">
+View example of he fluid breakout layout without an aside
+</a></div>
+
+### Uses
+
+A couple of examples of where this layout might be useful. Both examples include a section with the regular 12 column grid for comparison.
+
+A large table that would require truncation if fitted within the regular 12 column grid:
```

Comment:
Because this example relies on custom parameters passed to fluid layout it would be good to mention that in the docs, so it is clear that this example required different params than default built in layout.

Reply:
added

Reply:
reworded